packages/locale/res/translations/it.json

```diff
@@ -0,0 +1,467 @@
+    ["Failed to scan QR code. Error: ", "Scansione del codice QR non riuscita. Errore: "],
+    ["Please enter your email address and new master password.", "Inserisci il tuo indirizzo email e la nuova password principale."],
+    ["New Master Password", "Nuova Password Principale"],
+    ["WARNING: Weak Password!", "ATTENZIONE: Password debole!"],
+    ["Repeat Master Password", "Ripetere la password principale"],
+    [
+        "IMPORTANT, READ CAREFULLY: Padloc is designed in a way that makes it impossible for us to access the data encrypted in any of your vaults even if we wanted to. While this is essential to ensuring the security of your data, it also has the following implications:",
+        "IMPORTANTE, LEGGERE ATTENTAMENTE: Padloc è stato progettato in un modo che rende impossibile per noi accedere ai dati crittografati in qualsiasi tuo archivio, anche volendo. Anche se questo è essenziale per garantire la sicurezza dei tuoi dati, ha anche le seguenti implicazioni:"
+    ],
+    ["Any data stored in your private vault can not be recovered and will be permantently lost.", "Tutti i dati memorizzati nel vostro archivio privato non possono essere recuperati e saranno persi in modo permanente."],
+    ["All your organization memberships will be suspended temporarily until confirmed by the organization owner.", "Tutti i membri della tua organizzazione saranno temporaneamente sospesi fino alla conferma da parte del proprietario dell'organizzazione."],
+    ["All members of organizations you own will be suspended temporarily until confirmed by you.", "Tutti i membri delle organizzazioni che possiedi saranno temporaneamente sospesi fino alla conferma da parte tua"],
+    ["Please enter a master password!", "Inserisci una password principale!"],
+    ["You didn't repeat your master password correctly. Try again!", "Non hai ripetuto correttamente la password principale. Riprova!"],
+    [
+        "The password you entered is weak which makes it easier for attackers to break the encryption used to protect your data. Try to use a longer password or include a variation of uppercase, lowercase and special characters as well as numbers!",
+        "La password inserita è debole, il che rende più facile per gli aggressori annullare la crittografia utilizzata per proteggere i dati. Prova a usare una password più lunga o includi una variazione di caratteri maiuscoli, minuscoli e speciali oltre ai numeri!"
+    ],
+    ["Choose Different Password", "Scegli una password diversa"],
+    ["Use Anyway", "Usa comunque"],
+    ["WARNING: Weak Password", "ATTENZIONE: Password debole"],
+    ["To complete the account recovery process, please enter the confirmation code sent to your email address!", "Per completare il processo di recupero dell'account, inserisci il codice di conferma inviato al tuo indirizzo email!"],
+    ["Account recovery successful!", "Recupero account eseguito con successo!"],
+    ["Report Errors", "Report Errori"],
```

Comment:
Should this have been `Segnalari Errori`?

Reply:
This was a tricky one, because if we talk about the action to report, yes it's 'Segnala Errori', if instead is the report of errors like list of errors, then it can be like this

Reply:
Thanks for the clarification. This is about the action, as it shows as a title of the dialog to report the errors to Padloc, so I think it fits in the first option. Since it's also in the infinitive form "To Report", I think "Segnalari" might make more sense than "Segnala" (which is second person of the singular form, I believe), but I'll leave that up to you.

Reply:
Thank you for explanation! So since it's the action, it is between 'Segnalare Errori' or 'Segnala Errori', the infinitive form (Segnalare) in general is a bit wierd when you use it on an action on the web. Indeed, all the actions on mobile & web apps that have this feature use the 2nd person

Reply:
Va bene! A me sembra buono.